mailbox/api/src/main/java/org/apache/james/mailbox/MailboxListener.java

```diff
@@ -56,6 +61,61 @@
      *            not null
      */
     void event(Event event);
+    
+    interface QuotaEvent extends Event {
+        QuotaRoot getQuotaRoot();
+    }
+
+    class QuotaUsageUpdateEvent implements QuotaEvent, Serializable {
```

Comment:
`Updated` IMO, event are from the past. Not sure it's consistent with existing code.

Reply:
That was your proposal (cf comment above) but yes, ok.